tensorflow/lite/python/op_hint.py

```diff
@@ -583,7 +583,7 @@ def aggregate_and_return_name_for_input(self, out_graphdef):
 
     In particular, if you have 4 inputs to a hint stub, this will be the
     node that you can use as an output. I.e. you have 4 timesteps from a
-    static rnn, then a fused UnidriecitonalLSTM will expect 1 input with
+    static rnn, then a fused UndirectionalLSTM will expect 1 input with
```

Comment:
this does not seem to be changed?

this should be UnidirectionalLSTM

Reply:
You are right. sorry for my overlooking.